Files/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -178,11 +179,10 @@
                                 <Image
                                     Grid.RowSpan="3"
                                     Grid.Column="0"
+                                    x:Name="DriveThumbnailImage"
                                     Width="40"
                                     Height="40"
-                                    HorizontalAlignment="Center"
-                                    x:Phase="1"
```

Comment:
Shouldn't we still use `x:Phase`?

Reply:
@yaichenbaum Yes. I've added it back.

Reply:
@yaichenbaum Actually, I remembered why it was correctly removed. x:Phase must be used in conjunction with x:Bind.